### PR TITLE
Added gallery to company page & admin page

### PR DESCRIPTION
**Pull request by megaboy101**
This adds a gallery to a companies page if they have added photos via the admin dashboard. Both adding and deleting photos can be handled via the admin page. If company does not yet have any photos uploaded, the gallery will not appear.

Note: Adding more than 4-5 photos at a time may cause errors, due to hitting the Firebase upload size limit for a single request. This can be mitigated if photos are added in small batches, but it is something to be aware of.

### CODE REVIEW

File: src/components/AdminEditCompanyPage.js [hunk 1]
```diff
@@ -192,22 +242,69 @@ export const AdminEditCompanyPage = ({
...
+        if (newCover) {
+          update.coverPath = newCover.ref.fullPath;
+        }
+
+        if (newLogo) {
+          update.logoPath = newLogo.ref.fullPath;
+        }
+
+        if (photoUrls && photoUrls.length > 0) {
+          console.log(photoUrls);
```
Comment (Smudded) [inline src/components/AdminEditCompanyPage.js]: probably want to get rid of this log

File: src/components/AdminEditCompanyPage.js [hunk 2]
```diff
@@ -192,22 +242,69 @@ export const AdminEditCompanyPage = ({
...
-          // only set the fields we changed so we don't overwrite someone
-          // else
-          if (newCover) {
-            update.coverPath = newCover.ref.fullPath;
-          }
-          if (newLogo) {
-            update.logoPath = newLogo.ref.fullPath;
-          }
-          return doc.current.update(update);
+      // 2: Fetch storage URL's for linking photo's in database
```
Comment (Smudded) [inline src/components/AdminEditCompanyPage.js]: photos* :P

**Comment by Smudded** [comment]
Hm, can we split up each photo upload into it's own request so we don't run in to that limitation?

**Comment by megaboy101** [comment]
I took care of the minor edits you wrote down.

With regards to the upload limits, when I looked back through the code it looks like the photos are in fact setup to upload sequentially. Additionally, I can't seem to be able to reproduce the issue anymore. I'm safely able to add upwards of 30 images without seeing any issues. I also checked if it had to do with the size of a single image rather than the size of all of them combined, but I was able to upload a 10MB image just fine, and I don't think we will be uploading images larger than that anytime soon.

**Pull request merged**